client/lib/plans/features-list.js

```diff
@@ -695,7 +695,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME ]: {
 		getSlug: () => constants.FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Real-time{{/strong}} Backups', {
+			i18n.translate( '{{strong}}Real-time{{/strong}} backups', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.